src/server.js

```diff
@@ -31,23 +31,34 @@ const Config = require('./lib/config.js')
 const Logger = require('@mojaloop/central-services-shared').Logger
 const Plugins = require('./plugins')
 const RequestLogger = require('./lib/requestLogger')
-const ParticipantEndpointCache = require('@mojaloop/central-services-shared').Util.Endpoints
+const CSUtil = require('@mojaloop/central-services-shared').Util
+const ParticipantEndpointCache = CSUtil.Endpoints
 const Migrator = require('./lib/migrator')
 const ErrorHandler = require('@mojaloop/central-services-error-handling')
 const Boom = require('@hapi/boom')
+const models = require('./models')
+const domain = require('./domain')
+const Central = { ErrorHandler }
+const HeaderValidator = require('./lib/validation')
 
 const connectDatabase = async () => {
-  return await Db.connect(Config.DATABASE_URI)
+  await Db.connect(Config.DATABASE_URI)
 }
 
-const openAPIOptions = {
-  api: Path.resolve(__dirname, './interface/api_swagger.json'),
-  handlers: Path.resolve(__dirname, './handlers')
+const serviceType = {
```

Comment:
This can be moved to global enum on central-services-shared

Reply:
I'll happily do so if you think it'll be used elsewhere?